### PR TITLE
Do not ask for appname twice

### LINKED ISSUE #133: Don't ask for required input
app-name is a required param. Yet the slush generator still asks to confirm it. Make it optional, and if provided, skip asking to confirm..

### PR DESCRIPTION
**Pull request by patrickmcelwee**
Fixes #133

Also, make it optional on the command line

### CODE REVIEW

**Comment by grtjn** [comment]
needs a rebase

**Comment by patrickmcelwee** [comment]
ok, resolved conflicts

File: slushfile.js [hunk 1]
```diff
@@ -291,16 +287,25 @@ gulp.task('init', ['checkForUpdates'], function (done) {
...
+  var prompts = [
     {type: 'input', name: 'nodePort', message: 'Node app port?', default: 9070},
     {type: 'input', name: 'appPort', message: 'MarkLogic App/Rest port?', default: 8040},
     {type: 'input', name: 'xccPort', message: 'XCC port?', default:8041},
     {type: 'list', name: 'mlVersion', message: 'MarkLogic version?', choices: ['8','7', '6', '5'], default: 0},
     {type: 'confirm', name: 'includeEsri', message: 'Include ESRI Maps?', default: false}
-  ],
-  function (answers) {
-    answers.nameDashed = _.slugify(answers.name);
+  ]
```
Comment (grtjn) [inline slushfile.js]: missing semi-colon..

**Comment by grtjn** [comment]
Otherwise good to go. Worked smoothly even without the semi-colon..

**Comment by patrickmcelwee** [comment]
yep ... thanks for catching that. Added the semicolon

**Pull request merged**